beta/nncf/tensorflow/graph/converter.py

```diff
@@ -133,27 +142,19 @@ def convert_keras_model_to_nncf_graph(model: tf.keras.Model) -> NNCFGraph:
 
 
 def _get_nncf_graph_from_functional(model: tf.keras.Model) -> NNCFGraph:
+    #pylint:disable=too-many-statements
+    #pylint:disable=too-many-branches
```

Comment:
Please, refactor code and resolve pylint errors

Reply:
Done

Reply:
Thanks, looks cool!